specification/columns/commitmentdiscountunit.md

```diff
@@ -0,0 +1,36 @@
+# Commitment Discount Unit
+
+Commitment Discount Unit is the string value representing the provider-specified measurement unit that corresponds to [Commitment Discount Quantity](#commitmentdiscountconsumedquantity) and [Commitment Discount Purchased Quantity](#commitmentdiscountpurchasedquantity) of a [*commitment discount*](#glossary:commitment-discount). The CommitmentDiscountUnit column is only applicable to *commitment discounts* and not [*negotiated discounts*](#glossary:negotiated-discount).
```

Comment:
This makes sense for usage commitments. What about spend commitments? This doesn't convey what unit I should use for those.

Reply:
You should hours but it's up to you

Reply:
FOCUS can't dictate the units right. Spend commitments I would think will use some sort of currency as the unit.